buoyant_cloud/manifest.json

```diff
@@ -0,0 +1,91 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dee4b74f-34b7-457e-98b1-7bb8306f2c18",
```

Comment:
I confirmed this matches the `published_app_uuid` of the developer platform app. Thanks for doing that step!